gadget/gadget.go

```diff
@@ -625,6 +615,7 @@ func (e *editionNumber) UnmarshalYAML(unmarshal func(interface{}) error) error {
 type Size uint64
 
 const (
+	SizeKiB = Size(1 << 10)
 	SizeMiB = Size(1 << 20)
```

Comment:
I wonder if these constants are not interesting enough to be declared project-wide. There are other size-related variables in the codebase multiplying values by 1024 or 1024 * 1024. (And in this case maybe plain MiB, GiB, etc are more idiomatic in Go than SizeMiB and SizeGiB for project-wide units.)

Reply:
It'll probably need some consideration and a follow up though. I briefly looked at the places where 1024 is used in multiplication, it's not necessarily obvious there would be benefit from using common values.